src/components/link/link.tsx

```diff
@@ -138,6 +153,15 @@ export class Link implements InteractiveComponent {
   /** the rendered child element */
   private childEl: HTMLAnchorElement | HTMLSpanElement;
 
+  /**
```

Comment:
This doc/comment block is not needed. The handler name already expresses this information.